### PR TITLE
QuickHash improvement

### PR DESCRIPTION
**Pull request by saundefined**

### CODE REVIEW

File: reference/quickhash/quickhashinthash.xml [hunk 1]
```diff
@@ -14,10 +14,10 @@
    <para>
     This class wraps around a hash containing integer numbers, where the
     values are also integer numbers. Hashes are also available as
-    implementation of the ArrayAccess interface.
+    implementation of the <interfacename>ArrayAccess</interfacename> interface.
    </para>
    <para>
-    Hashes can also be iterated over with foreach as the Iterator interface is
+    Hashes can also be iterated over with <function>foreach</function> as the <interfacename>Iterator</interfacename> interface is
```
Comment (cmb69) [inline reference/quickhash/quickhashinthash.xml:20]: While there are apparently two occurrences of `<function>foreach</function>` somewhere else in the manual, I don't think this is right. `foreach` is not even like a function, but a language construct. As such, `<literal>function</literal>` is more appropriate; could optionally embed in a link element: `<link linkend="control-structures.foreach"><literal>foreach</literal></link>`.

File: reference/quickhash/quickhashintset.xml [hunk 2]
```diff
@@ -15,7 +15,7 @@
     This class wraps around a set containing integer numbers.
    </para>
    <para>
-    Sets can also be iterated over with foreach as the Iterator interface is
+    Sets can also be iterated over with <function>foreach</function> as the <interfacename>Iterator</interfacename> interface is
```
Comment (cmb69) [inline reference/quickhash/quickhashintset.xml:18]: ditto

File: reference/quickhash/quickhashintstringhash.xml [hunk 3]
```diff
@@ -14,10 +14,10 @@
    <para>
     This class wraps around a hash containing integer numbers, where the
     values are strings. Hashes are also available as implementation of the
-    ArrayAccess interface.
+    <interfacename>ArrayAccess</interfacename> interface.
    </para>
    <para>
-    Hashes can also be iterated over with foreach as the Iterator interface is
+    Hashes can also be iterated over with <function>foreach</function> as the <interfacename>Iterator</interfacename> interface is
```
Comment (cmb69) [inline reference/quickhash/quickhashintstringhash.xml:20]: ditto

File: reference/quickhash/quickhashstringinthash.xml [hunk 4]
```diff
@@ -14,10 +14,10 @@
    <para>
     This class wraps around a hash containing strings, where the
     values are integer numbers. Hashes are also available as
-    implementation of the ArrayAccess interface.
+    implementation of the <interfacename>ArrayAccess</interfacename> interface.
    </para>
    <para>
-    Hashes can also be iterated over with foreach as the Iterator interface is
+    Hashes can also be iterated over with <function>foreach</function> as the <interfacename>Iterator</interfacename> interface is
```
Comment (cmb69) [inline reference/quickhash/quickhashstringinthash.xml:20]: ditto

**Review by cmb69 — COMMENTED** [review]
Nice improvement, thank you! The `<function>foreach</function>` issue should be adressed; otherrwise this is fine.

File: reference/quickhash/quickhashinthash.xml (hunk 1 above)
Comment (saundefined) [reply reference/quickhash/quickhashinthash.xml]: Ah, exactly! Thank you!

Comment (salathe) [reply reference/quickhash/quickhashinthash.xml]: Note: there is the `&foreach;` entity, defined in `language-snippets.ent` as:
```xml
<!ENTITY foreach '<link xmlns="http://docbook.org/ns/docbook" linkend="control-structures.foreach">foreach</link>'>
```

**Pull request merged**